src/rust/src/x509/certificate.rs

```diff
@@ -233,7 +233,7 @@ impl Certificate {
                     .collect();
                 if filtered_extensions.len() == ext_count {
                     return Err(PyAsn1Error::from(pyo3::exceptions::PyValueError::new_err(
-                        "Could not find SCT list extension in TBS certificate",
+                        "Could not find SCT list extension in TBS precertificate",
```

Comment:
```suggestion
                        "Could not find pre-certificate SCT list extension",
```

Here's my alternate suggestion